ocs_ci/deployment/hosted_cluster.py

```diff
@@ -477,6 +488,8 @@ def deploy_dependencies(
             self.deploy_lb()
         if download_hcp_binary:
             self.update_hcp_binary()
+        if deploy_mce:
```

Comment:
we should probably check, if deploy_acm_hub: True, leaving ACM as a general option
```suggestion
        if deploy_mce and not deploy_acm_hub:
```

Reply:
Addressed in latest commit.